source/command-logging-and-monitoring/tests/logging/command.yml

```diff
@@ -52,13 +52,14 @@ tests:
             component: command
             data:
               message: "Command succeeded"
+              databaseName: *databaseName
```

Comment:
Noted that the schema defines `data` as an `object` ([here](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/schema-1.13.json#L563)), so a new field here has no impact on the schema version.



Check unified test format files against schema / build (pull_request)